runner/config.go

```diff
@@ -237,6 +238,10 @@ func defaultConfig() Config {
 		Color:       color,
 		Log:         log,
 		Misc:        misc,
+		Screen: cfgScreen{
+			ClearOnRebuild: false,
+			KeepScroll:     false,
```

Comment:
I'd argue default should be `true` because a full clear is destructive to terminal history and generally not what people would want.

Reply:
I see your point.